tests/test_systems/test_spectrometers/test_HyperspectralImager.py

```diff
@@ -187,3 +187,14 @@ def test_get_pointing_accuracy_constraint():
     LOG.info(result.to(unit.degree))
 
     assert result.unit == unit.rad
+
+
+def test_get_ground_target_error():
+    """Test get_ground_target_error method."""
+
+    spectrometer = HyperspectralImager()
+
+    result = spectrometer.get_ground_target_error(550 * unit.km, 30 * unit.deg)
+    LOG.info(result)
+
+    assert result.unit == unit.km
```

Comment:
We only care about the base unit that comes out of the methods. We conduct such a check like this:
```suggestion
    assert result.decompose().unit == unit.m
```